e2e_test/batch/aggregate/selective_distinct_agg.slt.part

```diff
@@ -4,6 +4,21 @@ SET RW_IMPLICIT_FLUSH TO true;
 statement ok
 create table t(v1 int, v2 int, v3 int)
 
+query I
+select count(*), count(distinct v2) from t;
+----
+0 0
+
+query I
+select count(*), count(distinct v2) from t group by v3;
+----
+(empty)
```

Comment:
`(empty)` means an empty string rather than 0 rows. Just remove this line to fix e2e sqllogictest.